### PR TITLE
[zh] Sync controllers/cron-jobs.md

### PR DESCRIPTION
**Pull request by her-cat**
There seems a typo here, It should be `cron-job-limitations`.

### CODE REVIEW

**Comment by windsonsea** [comment]
Seems this file has some more in-synced content with the en upstream, which you can find by running this command on your `main` branch. Hope you can include those differences into this PR, thanks.

```
./scripts/lsync.sh content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md
```
The output will be:
```diff
diff --git a/content/en/docs/concepts/workloads/controllers/cron-jobs.md b/content/en/docs/concepts/workloads/controllers/cron-jobs.md
index 2ffdda8b34..cf9bddf694 100644
--- a/content/en/docs/concepts/workloads/controllers/cron-jobs.md
+++ b/content/en/docs/concepts/workloads/controllers/cron-jobs.md
@@ -4,6 +4,9 @@ reviewers:
 - soltysh
 - janetkuo
 title: CronJob
+api_metadata:
+- apiVersion: "batch/v1"
+  kind: "CronJob"
 content_type: concept
 description: >-
   A CronJob starts one-time Jobs on a repeating schedule.
@@ -158,12 +161,17 @@ When `.spec.suspend` changes from `true` to `false` on an existing CronJob witho
 
 ### Jobs history limits
 
-The `.spec.successfulJobsHistoryLimit` and `.spec.failedJobsHistoryLimit` fields are optional.
-These fields specify how many completed and failed Jobs should be kept.
-By default, they are set to 3 and 1 respectively.  Setting a limit to `0` corresponds to keeping
-none of the corresponding kind of Jobs after they finish.
+The `.spec.successfulJobsHistoryLimit` and `.spec.failedJobsHistoryLimit` fields specify
+how many completed and failed Jobs should be kept. Both fields are optional.
 
-For another way to clean up Jobs automatically, see [Clean up finished Jobs automatically](/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically).
+* `.spec.successfulJobsHistoryLimit`: This field specifies the number of successful finished
+jobs to keep. The default value is `3`. Setting this field to `0` will not keep any successful jobs.
+
+* `.spec.failedJobsHistoryLimit`: This field specifies the number of failed finished jobs to keep.
+The default value is `1`. Setting this field to `0` will not keep any failed jobs.
+
+For another way to clean up Jobs automatically, see
+[Clean up finished Jobs automatically](/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically).
```

**Comment by her-cat** [comment]
Please take a look. @windsonsea

**Comment by her-cat** [comment]
I solved these problems, please take a look again, thanks. @windsonsea

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -310,18 +316,28 @@ When `.spec.suspend` changes from `true` to `false` on an existing CronJob witho
...
 ### 任务历史限制   {#jobs-history-limits}
 
-`.spec.successfulJobsHistoryLimit` 和 `.spec.failedJobsHistoryLimit` 字段是可选的。
-这两个字段指定应保留多少已完成和失败的 Job。
-默认设置分别为 3 和 1。将限制设置为 `0` 代表相应类型的 Job 完成后不会保留。
+`.spec.successfulJobsHistoryLimit` 和 `.spec.failedJobsHistoryLimit`
+字段指定应保留多少已完成和失败的 Job。这两个字段都是可选的。
+
+* `.spec.successfulJobsHistoryLimit`：此字段指定要保留多少成功完成的 Job。默认值为 `3`。
+将此字段设置为 `0` 意味着不会保留任何成功的 Job。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md:337]:
```suggestion
  将此字段设置为 `0` 意味着不会保留任何成功的 Job。
```

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -310,18 +316,28 @@ When `.spec.suspend` changes from `true` to `false` on an existing CronJob witho
...
+
+* `.spec.failedJobsHistoryLimit`：此字段指定要保留多少失败完成的 Job。默认值为 `1`。
+将此字段设置为 `0` 意味着不会保留任何失败的 Job。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md:340]:
```suggestion
  将此字段设置为 `0` 意味着不会保留任何失败的 Job。
```

**Review by windsonsea — COMMENTED** [review]
Thanks, some nits though the en upstream has the same issue.

/approve

Tips: You can amend your edit after the initial commit to ensure that your PR only has one commit like:
```
git add .
git commit --amend --no-edit
git push -f
```

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md (hunk 2 above)
Comment (windsonsea) [reply content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md]: Two spaces there, not one. You can copy and paste it.

**Review by ydFu — APPROVED** [review]
/lgtm
/approve

**Pull request merged**